src/include/pbs_license.h

```diff
@@ -49,6 +49,8 @@ extern "C" {
 #define ND_LIC_TYPE_float 	'f'
 #define ND_LIC_TYPE_unlic	'u'
 #define ND_LIC_TYPE_cloud	'c'
+#define ND_LIC_locked_str	"l"
+#define ND_LIC_cloud_str	"c"
```

Comment:
Please align this with the previous macros.

Reply:
I'm using a tab in both the cases and it appears fine in Firefox. Can you check using some other browser as it appears to be rendering issue?